### PR TITLE
fix #213, Quart extension

### LINKED ISSUE #213: Add Quart extension
https://gitlab.com/pgjones/quart

### PR DESCRIPTION
**Pull request by wwwjfy**

### CODE REVIEW

File: tests/test_quart.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import sys
+
+# Quart only supports Python 3.6 or later
+if sys.version_info >= (3, 6):
```
[the remark this answers is not in the document]
Comment (gyermolenko) [reply tests/test_quart.py]: please take a look at [skipif](https://docs.pytest.org/en/2.9.0/skipping.html#id1)
Does it solve your case?

[the remark this answers is not in the document]
Comment (wwwjfy) [reply tests/test_quart.py]: Thanks! It looks nicer now.
The reported coverage falls down though

**Comment by wwwjfy** [comment]
the coverage fix is here: https://github.com/fantix/gino/pull/244

**Comment by fantix** [comment]
Yes! Please go ahead merge it.

**Pull request merged**